src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/ldap/OpenLdapUnsafeMethods.cs

```diff
@@ -0,0 +1,192 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+
+namespace System.DirectoryServices.Protocols
+{
+    internal class OpenLDAP
```

Comment:
This should be under src/libraries/Common/src/Interop/Unix; and the files should all be in the global namespace as

```suggestion
internal partial static class Interop
{
    internal partial static class OpenLDAP
```

https://github.com/axelheer/dotnet-corefx/blob/master/Documentation/coding-guidelines/interop-guidelines.md

Reply:
sounds good. Do you want me to also move the win32ldap calls to be under it's respective common folder as well? For openldap ones I only followed what we were doing already in this library for wldap32 so if I'm moving ones it probably makes sense to move the others too.

Reply:
Yep, you just beat me to that assertion :)

Reply:
Also, one of the interop guidelines is to use more granular files.  While we started with a "one method per file" approach, we've softened to "things you need together".

E.g. it might make sense for all of the LDAP methods to be in `src/libraries/Common/src/Interop/Unix/OpenLDAP/Interop.Ldap.cs` and all of the BER ones to be in `src/libraries/Common/src/Interop/Unix/OpenLDAP/Interop.Ber.cs`.  But it's probably not "all of this is one file"; even if they're currently used together.